app/views/resources/edit.html.erb

```diff
@@ -1,3 +1,5 @@
 <%= render partial: "partials/form_structure", locals: {model_name: "resource",
                                                         object: @resource,
                                                         action: "edit"} %>
+
+<%= link_to "Return to this competency's resources ", resources_path(:competency_id => @competency_id), class: "waves-effect waves-light btn"  %>

```

Comment:
Also just a suggestion on the UI side should we include like a back icon for clarity
like a '<' icon left of "Return to this competency's resources "